### ISSUE TITLE
Discover the path MTU to be used in OVN route agent driver.

### ISSUE THREAD

**Issue by aswinsuryan** [submariner-io/submariner#3097]
We use hardcode MTU values in [gateway_dataplane.go](https://github.com/submariner-io/submariner/blob/devel/pkg/routeagent_driver/handlers/ovn/gateway_dataplane.go#L97).

This needs to be enhanced to use the real path MTU calculated using tools like tracepath.

This is required as described here https://github.com/submariner-io/submariner/issues/1022

**What would you like to be added**:

**Why is this needed**:

**Comment by yboaron** [comment]
I guess we should retry checking with mss clamping rule to pmtu first

**Comment by dfarrell07** [comment]
I don't think this is a prio currently, or we don't have dev cycles for it, but we can re-open if it becomes an issue for someone.
